Sources/Widgets/Widgets3D/LineWidget/behavior.js

```diff
@@ -0,0 +1,259 @@
+import macro from 'vtk.js/Sources/macro';
+
+const MAX_POINTS = 2;
+
+export default function widgetBehavior(publicAPI, model) {
+  model.classHierarchy.push('vtkDistanceWidgetProp');
+  let isDragging = null;
+  const direction = [0, 0, 0];
+
+  // --------------------------------------------------------------------------
+  // Display 2D
+  // --------------------------------------------------------------------------
+
+  publicAPI.setDisplayCallback = (callback) =>
+    model.representations[0].setDisplayCallback(callback);
+
+  // --------------------------------------------------------------------------
+  // Interactor events
+  // --------------------------------------------------------------------------
+
+  function ignoreKey(e) {
+    return e.altKey || e.controlKey || e.shiftKey;
+  }
+
+  function updateConeDirection(behavior, callData) {
+    let bv = behavior;
```

Comment:
Not sure that it's necessary.

Reply:
Is there any other way to properly avoid Prettier messages like 'assigning value to function parameter', other than defining variables in local scope like i did here?

Reply:
We can leave it as it for now.